include/openthread/types.h

```diff
@@ -478,6 +478,17 @@ typedef struct otMessageInfo
     const void *mLinkInfo;
 } otMessageInfo;
 
+/**
+ * This enumeration defines the Openthread message priority levels
+ *
+ */
+typedef enum otMessagePriority {
+    OT_MESSAGE_PRIORITY_HIGH     = 0, ///< High priority level. It is reserved for internal network control messages.
```

Comment:
How about not defining `OT_MESSAGE_PRIORITY_HIGH`  in public `types.h` header file? Specially since it's restricted for internal use only so users of OpenThread do not need to know about this level and this constant.